lib/SPIRV/OCLTypeToSPIRV.cpp

```diff
@@ -1,4 +1,4 @@
-﻿//===- OCLTypeToSPIRV.cpp - Adapt types from OCL for SPIRV ------*- C++ -*-===//
+//===- OCLTypeToSPIRV.cpp - Adapt types from OCL for SPIRV ------*- C++ -*-===//
```

Comment:
Thanks. I guess the file was somehow messed up before. We'd better keep all files in ASCII.